spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'menu.rb'
+describe Menu do
+  subject(:menu) { described_class.new }
+  context "#initialize" do
+    it 'create a list if dishes' do
+      menu = Menu.new
+      expect(menu.dishes).to eq described_class::DISHES_LIST
+    end
+  end
+
+  context "#print_menu" do
+    it 'prints the menu' do
+      menu = Menu.new
+      show_menu = menu.dishes.each{ |dish, price| "#{dish}: £#{price}, " }
```

Comment:
The test would be more robust if you wrote out the string you meant. What if there were a bug in this line, or in `.dishes`?